internal/cmd/commands/server/server.go

```diff
@@ -157,6 +157,14 @@ func (c *Command) Run(args []string) int {
 			return 1
 		}
 	}
+	if cfg.GoogleWorkspace == nil {
+		c.UI.Error("error validating config: google_workspace block is required")
+		return 1
+	}
+	if cfg.GoogleWorkspace.Domain == "" {
+		c.UI.Error("error validating config: google_workspace domain is required")
```

Comment:
```suggestion
		c.UI.Error("error validating config: Google Workspace Domain is required")
```

We might want to be consistent with how we show these errors for invalid server config. For reference: https://github.com/hashicorp-forge/hermes/pull/261/files#diff-39d8b05de61be8fba5e909420b32a4182700e95200188a0be03efccc858000f0R200-R202

Reply:
Good call, I'll actually move this under that part you referenced. All of this needs to be refactored at some point, but that'll be more consistent for now.

Reply:
Sounds good to me!

Reply:
Done!